src/bounding_hierarchy.rs

```diff
@@ -14,6 +14,13 @@ pub trait BoundingHierarchy {
     /// use bvh::aabb::{AABB, Bounded};
     /// use bvh::bounding_hierarchy::BoundingHierarchy;
     /// use bvh::nalgebra::{Point3, Vector3};
+    /// # use bvh::bounding_hierarchy::BHShape;
+    /// #
+    /// # impl BHShape for AABB {
```

Comment:
This `impl` is invalid. Unfortunately we need to define an explicit structure and define BVHShape for it.
And also unfortunately we can't hide or deduplicate this boilerplate code using utilities or macros, because doctest are executed in an integration test environment.